<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-10R8ABVG" failures="1" tests="3" name="smoke" time="54.864" errors="0" timestamp="2024-01-03T16:15:22 IST">
  <testcase classname="com.sevenrmartsupermarket.tests.PushNotificationsTest" name="verifyPushNotificationMessage" time="6.326"/>
  <testcase classname="com.sevenrmartsupermarket.tests.ManageDeliveryBoyTest" name="verifyCreatingNewDeliveryBoy" time="25.99">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 0 out of bounds for length 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at com.sevenrmartsupermarket.tests.ManageDeliveryBoyTest.verifyCreatingNewDeliveryBoy(ManageDeliveryBoyTest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- verifyCreatingNewDeliveryBoy -->
  <testcase classname="com.sevenrmartsupermarket.tests.ManageDeliveryBoyTest" name="verifyDeliverBoyEdit" time="5.053"/>
</testsuite> <!-- smoke -->
